android/filament-utils-android/src/main/java/com/google/android/filament/utils/ModelViewer.kt

```diff
@@ -110,7 +110,7 @@ class ModelViewer(val engine: Engine) : android.view.View.OnTouchListener {
         view.scene = scene
         view.camera = camera
 
-        assetLoader = AssetLoader(engine, MaterialProvider(engine), EntityManager.get())
+        assetLoader = AssetLoader(engine, UbershaderLoader(engine), EntityManager.get())
```

Comment:
This Java was badly named, it was actually an ubershader loader. After this PR lands, it will be easy to add a feature to ModelViewer that allows clients to pass in a custom provider object.